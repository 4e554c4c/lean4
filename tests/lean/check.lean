--

#check and.intro
#check or.elim
#check Eq
#check Eq.rec
