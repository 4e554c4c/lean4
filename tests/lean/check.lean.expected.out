And.intro : ?M_1 → ?M_2 → ?M_1 ∧ ?M_2
Or.elim : ?M_1 ∨ ?M_2 → (?M_1 → ?M_3) → (?M_2 → ?M_3) → ?M_3
Eq : ?M_1 → ?M_1 → Prop
Eq.rec : ?M_3 ?M_2 _ → Π {a : ?M_1} (t : ?M_2 = a), ?M_3 a t
